spec.txt

```diff
@@ -3702,6 +3727,16 @@ The following rules define [block quotes]:
     marker] to the beginning of each line in *Ls*
     is a [block quote](#block-quotes) containing *Bs*.
 
+    To prepend a [complete block quote marker] to a line, calculate its
+    indentation without the block quote marker at the current column (N1),
+    then insert `>` at the start of the line, then calculate its indentation at
+    the same column (N2, which will equal N1 unless the line starts with a tab),
+    then add M=N1+1-N2 spaces of indentation (if M is negative, remove spaces).
```

Comment:
This sentence calls for adding or removing spaces after adding the `>`, so the resulting line has M+N2 spaces after the `>`. Though M might be negative, M+N2 equals N1+1, which is a positive number.